src/OperatorStateRetriever.sol

```diff
@@ -178,4 +178,25 @@ contract OperatorStateRetriever {
         }
         return quorumBitmaps;
     }
+
+    function getBatchOperatorId(
```

Comment:
These two methods will revert if any of the ID/address doesn't exist?

Reply:
as written, these methods currently will _not_ revert, but instead return `bytes32(0)` as the ID of any operator who has never registered, or `address(0)` for any operatorID that has never been used.

relevant code:
https://github.com/Layr-Labs/eigenlayer-middleware/blob/feat/batch-operatorid-view/src/BLSApkRegistry.sol#L275-L276
https://github.com/Layr-Labs/eigenlayer-middleware/blob/feat/batch-operatorid-view/src/RegistryCoordinator.sol#L796-L804

@jianoaix what would your preferred behavior here be?

Reply:
Yep, that's more desirable! Could we put this in the comment of the function?

Reply:
No they will just return a 0 value at that index